Database/index.spec.ts

```diff
@@ -1,8 +1,14 @@
 import * as storage from "../index"
 interface Item {
 	value: number
+	level: number
+	groups: any[]
+	name: string
+	created: string
+	changed: string
+	address: Record<string, any>
 }
-type Layout = { archive: { items: Item } }
+type Layout = { archive: { items: Partial<Item> } }
```

Comment:
Instead of doing Partial I  would recommend making the properties you want to be optional optional.